### PR TITLE
Update geofabrics to work with the latest version of geoapis

### PR DESCRIPTION
**Pull request by rosepearson**
Update geofabrics to work with the latest version of geoapis (it includes a change to the lidar.OpenTopography interface):
 - [X] Make code change
 - [x] Update tests 
   - [X] Update / create new tests
   - [X] Ensure these tests have the expected behavour
   - [x] Test locally and ensure tests are passing
 - [x] Ensure tests passing on GH Actions
 - [X] Update documentation
   - [X] Doc strings
   - [X] Wiki

### CODE REVIEW

**Comment by rosepearson** [comment]
Kia ora @Martin20494 do you mind reviewing the PR please as this includes a change required to ensure compatability with the latest update to geoapis that we will be using later today :) 
Ngā mihi
Rose

File: src/geofabrics/processor.py [hunk 1]
```diff
@@ -239,8 +239,8 @@ def get_lidar_file_list(self, data_service) -> dict:
                 "'file_paths' in the instruction file if you are going to use an API - like 'open_topography'"
 
             # download from OpenTopography - then get the local file path
-            self.lidar_fetcher = geoapis.lidar.OpenTopography(self.catchment_geometry.catchment,
-                                                              self.get_instruction_path('local_cache'),
+            self.lidar_fetcher = geoapis.lidar.OpenTopography(cache_path=self.get_instruction_path('local_cache'),
+                                                              search_polygon=self.catchment_geometry.catchment,
```
Comment (Martin20494) [inline src/geofabrics/processor.py:243]: I have a look on this already.

**Review by Martin20494 — APPROVED** [review]
Happy to land it

File: src/geofabrics/processor.py (hunk 1 above)
Comment (rosepearson) [reply src/geofabrics/processor.py:243]: Thanks Martin!

**Pull request merged**